test/e2e/common_test.go

```diff
@@ -945,7 +945,7 @@ func (p *PodmanTestIntegration) PodmanAsUser(args []string, uid, gid uint32, cwd
 // RestartRemoteService stop and start API Server, usually to change config
 func (p *PodmanTestIntegration) RestartRemoteService() {
 	p.StopRemoteService()
-	p.StartRemoteService()
+	p.StartRemoteService("info")
```

Comment:
> Quick & easy solution: s/info/version/. Super nice, very readable, but there are actual instances (containers.conf and network testing) where it's important to confirm 'podman info' settings on a test failure.

Why is info of any relevance here? Local doesn't clutter us with info output as well. Each test should already have enough information to debug it. I never needed the info output in remote tests.

Reply:
I was going for a middle ground. I've never used the `info` output in podman-remote tests, but that doesn't mean nobody does. I have no way to know.

And, good point: I did not try removing `delayForService`, nor even looking at git logs to see if its history is still relevant. I can do so next week.